StrictEmit/StrictEmit.csproj

```diff
@@ -46,6 +46,8 @@
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta004" PrivateAssets="all" />
     <PackageReference Include="JetBrains.Annotations" Version="11.1.0" PrivateAssets="all" />
     <PackageReference Include="JetBrainsAnnotations.Fody" Version="2.0.0" PrivateAssets="all" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
```

Comment:
There's no need to conditionally exclude the package. The framework will resolve the correct dependecy at compile time.

Reply:
The minium .NET Framework version for the package `System.Reflection.Emit` is 4.5, 
thus it gives an error if this is not excluded from .NET Framework 4.0.